tests/main/interfaces-cups-control/task.yaml

```diff
@@ -74,4 +72,4 @@ execute: |
         echo "Expected error with plug disconnected"
         exit 1
     fi
-    MATCH "scheduler not responding" < print.error
+    MATCH "(scheduler not responding|No default destination)" < print.error
```

Comment:
Fwiw, this seems to be needed because the `test-snapd-cups-control-consumer` moved to `base: core20` very recently.